packages/app/components/card/list-card.tsx

```diff
@@ -0,0 +1,356 @@
+import { Suspense, useMemo, useCallback, ReactNode } from "react";
+import {
+  Platform,
+  StyleProp,
+  useWindowDimensions,
+  ViewStyle,
+} from "react-native";
+
+import { ResizeMode } from "expo-av";
+import { Link, LinkProps } from "solito/link";
+
+import {
+  PressableScale,
+  Props as PressableScaleProps,
+} from "@showtime-xyz/universal.pressable-scale";
+import { Skeleton } from "@showtime-xyz/universal.skeleton";
+import { Text } from "@showtime-xyz/universal.text";
+import { View } from "@showtime-xyz/universal.view";
+
+import { Creator } from "app/components/card/rows/elements/creator";
+import { Title } from "app/components/card/rows/title";
+import { ClaimButton } from "app/components/claim/claim-button";
+import { ClaimedShareButton } from "app/components/claim/claimed-share-button";
+import { ErrorBoundary } from "app/components/error-boundary";
+import { ClaimedByReduced } from "app/components/feed-item/claimed-by";
+import { ListMedia } from "app/components/media";
+import { withMemoAndColorScheme } from "app/components/memo-with-theme";
+import { NFTDropdown } from "app/components/nft-dropdown";
+import { useCreatorCollectionDetail } from "app/hooks/use-creator-collection-detail";
+import { useNFTDetailByTokenId } from "app/hooks/use-nft-detail-by-token-id";
+import { linkifyDescription } from "app/lib/linkify";
+import { NFT } from "app/types";
+import { removeTags } from "app/utilities";
+
+import { breakpoints } from "design-system/theme";
+
+import { ContentTypeTooltip } from "../content-type-tooltip";
+import { NSFWGate } from "../feed-item/nsfw-gate";
+
+const isWeb = Platform.OS === "web";
+
+const RouteComponent = ({
+  children,
+  onPress,
+  ...rest
+}: (LinkProps | PressableScaleProps) & {
+  onPress: () => void;
+  children: ReactNode;
+}) => {
+  if (isWeb) {
+    return <Link {...(rest as LinkProps)}>{children}</Link>;
+  }
+  return (
+    <PressableScale onPress={onPress} {...(rest as PressableScaleProps)}>
+      {children}
+    </PressableScale>
+  );
+};
+
+const RouteComponentNative = ({
+  children,
+  onPress,
+  ...rest
+}: (LinkProps | PressableScaleProps) & {
+  onPress: () => void;
+  children: ReactNode;
+}) => {
+  if (isWeb) {
+    return <View tw="flex-1">{children}</View>;
+  }
+  return (
+    <PressableScale onPress={onPress} {...(rest as PressableScaleProps)}>
+      {children}
+    </PressableScale>
+  );
+};
+
+type Props = {
+  nft: NFT & { loading?: boolean };
+  numColumns?: number;
+  onPress?: () => void;
+  tw?: string;
+  variant?: "nft" | "activity" | "market";
+  href?: string;
+  showClaimButton?: Boolean;
+  sizeStyle?: { width: number; height: number };
+  style?: StyleProp<ViewStyle>;
+};
+
+function ListCard(props: Props) {
+  const { width } = useWindowDimensions();
+  const isLgWidth = width >= breakpoints["md"];
+
+  if (!isLgWidth) {
+    return <ListCardSmallScreen {...props} />;
+  }
+
+  return <ListCardLargeScreen {...props} />;
```

Comment:
not a blocker: i have seen stuff jumping as we use JS based logic for screen resolution mounting. We should probably start using tailwind's `hidden` in future.

Reply:
This was the logic of the old `Card`, I just optimized it a bit. We can keep optimizing in the future for sure.